public/views/pages/payments/stripe/checkout_session_completed_webhook.liquid

```diff
@@ -14,8 +14,23 @@ layout: null
     if object.valid
       echo object
     else
-      log object, type: 'ERROR: payments_stripe/checkout_session_completed_webhook object'
-      response_status 500
+      # Stripe will ping all endpoints so maybe we get another instance's ID
+      # and that's why the command will not valid.
+      # We should not send 500 status response in this case, because the request was valid,
+      # but it was not relevant in the current instance.
+
+      assign log_level = 'WARNING'
+      if object.errors.id contains 'cannot be blank'
```

Comment:
@hosszukalman Please test it manually or with unit test cause I can't see validation for `id` field in the command.
https://github.com/Platform-OS/pos-module-payments/blob/master/public/lib/commands/gateway_requests/receive/check.liquid

Reply:
Yes, it simply won't work, plus 'modules/payments_stripe/commands/checkout_session/complete' will always create `gateway_request` even though it shouldn't because we don't have that transaction in the db. Why not check if transaction exists before line 13?

Reply:
You were right. When I tested it worked but the error message came from the `modules/payments/commands/transactions/update_status` function, so the `gateway_request` was created. I updated the code.

Reply:
this should work. One more thing I have noticed is that `modules/payments_stripe/commands/checkout_session/complete` never updates `status` of `checkout_session` and I guess it should be updated with whatever comes by webhook

Reply:
I will check that bug too (but if I remember correctly @piotrze removed `checkout_session` table and now everything is stored in `gateway_request` table).
Otherwise I would check that bug in a separate PR because I think it's not related to this one. Do you agree @streflik?

Reply:
yes, not related